bugbot/rules/not_landed.py

```diff
@@ -232,6 +254,7 @@ def attachment_handler(attachments, data):
 
             if "phab" in res:
                 if res["phab"]:
+                    data[bugid]["dependencies"] = has_blocking_dependencies(attachment)
```

Comment:
Could you use a more descriptive name, e.g. `has_blocking_dependencies` or `is_blocked`?

Reply:
I've already changed the function from `search_dependencies()` to `has_blocking_dependencies()`. Are you referring to another function?

Reply:
I'm referring the the field, not the function

Reply:
Changed here: 1fd88b14fbca3ea97c512bcd300c5130a433ebac. Thanks for pointing that out!